src/igvfd/schemas/variant.json

```diff
@@ -0,0 +1,138 @@
+{
+    "title": "Variant",
+    "description": "Schema for submitting a variant.",
+    "$id": "/profiles/variant.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "ref",
+        "alt",
+        "locations"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "variantid": {
+            "title": "Variant ID",
+            "description": "An external ID for the variant like dbSNP.",
+            "comment": "This is currently only accepting rsIDs",
+            "type": "string",
+            "pattern": "rs[0-9]+",
+            "uniqueKey": true
+        },
+        "associated_gene": {
```

Comment:
I think that will probably end up as an array given some of the input from Friday, but lets only change it if there are other things that need changing